packages/core/src/utils/checkpoint.ts

```diff
@@ -2,9 +2,9 @@ export type Checkpoint = {
   blockTimestamp: number;
   chainId: number;
   blockNumber: number;
-  // Execution index of the log within the block.
-  // If undefined, the checkpoint includes all events in the block.
-  logIndex?: number | undefined;
+  transactionIndex: number;
+  eventType: number;
+  eventIndex: number;
```

Comment:
We should standardize naming for `eventIndex` - looks like there are some leftover `EXECUTION_INDEX` names in this file.